cinder/db/sqlalchemy/api.py

```diff
@@ -6470,7 +6470,7 @@ def purge_deleted_rows(context, age_in_days):
                     session.query(models.QualityOfServiceSpecs).filter(
                         and_(models.QualityOfServiceSpecs.specs_id.isnot(
                             None),
-                            models.QualityOfServiceSpecs.deleted == True,
+                            models.QualityOfServiceSpecs.deleted is True,
```

Comment:
Does SQLAlchemy support the `is`? I would expect this to be evaluated first and the result passed to the underlying function. I know SQLAlchemy overwrites `__eq__` for the `==` operator, but I haven't found anything about `is`.

Reply:
No, it doesn't. I've fixed this in the latest commit.